_episodes/03-create.md

```diff
@@ -541,7 +541,7 @@ ls: cannot access 'quotes.txt': No such file or directory
 > when we delete files, they are unlinked from the file system so that
 > their storage space on disk can be recycled. Tools for finding and
 > recovering deleted files do exist, but there's no guarantee they'll
-> work in any particular situation, since the computer may recycle the
```

Comment:
I think the particular is important here to highlight that *some* situations this may be the case.

Reply:
Addressed in the next commit